x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -492,3 +492,6 @@ export const DEFAULT_DETECTION_PAGE_FILTERS = [
     fieldName: 'host.name',
   },
 ];
+
+export const SECURITY_SOLUTION_ACTION_TRIGGER = 'security-solution-default';
```

Comment:
We should have a more specific name here, mentioning this is for cell actions at least (we may want to register other kinds of actions inside Security later on). What about `security-solution-default-cellActions` and `security-solution-timeline-cellActions`?